src/test/compile-fail/privacy-sanity.rs

```diff
@@ -9,8 +9,11 @@
 // except according to those terms.
 
 #![feature(optin_builtin_traits)]
+#![feature(immovable_types)]
 
-trait MarkerTr {}
+use std::marker::Move;
+
+trait MarkerTr: ?Move {}
```

Comment:
(likewise here, I am again curious as to whether this was a necessary change, or just an artifact of your methodology for updating the tests...)

Reply:
(I'll stop asking about the future instances of this pattern now, though.)